libs/perl.json

```diff
@@ -0,0 +1,7 @@
+[
+  {"id":"perl.Mojolicious","imports":["Mojolicious"],"name":"Mojolicious","repo":"mojolicious/mojo","tags":["web-framework"],"tech":["internet"],"status":"awaiting-model"},
+  {"id":"perl.Plack","imports":["Plack"],"name":"Plack","repo":"plack/Plack","tags":["web-framework"],"tech":["internet"],"status":"awaiting-model"},
+  {"id":"perl.Moose","imports":["Moose"],"name":"Moose","repo":"moose/Moose","tags":["metaprogramming"],"tech":["app-arch"],"status":"awaiting-model"},
+  {"id":"perl.Minilla","imports":["Minilla"],"name":"Minilla","repo":"tokuhirom/Minilla","tags":["metaprogramming"],"tech":["app-arch"],"status":"awaiting-model"},
+  {"id":"perl.Moo","imports":["Moo"],"name":"Moo","repo":"moose/Moo","tags":["metaprogramming"],"tech":["app-arch"],"status":"awaiting-model"}
```

Comment:
tags = oop? and maybe the same for moose? it would seem that meta is an extra there